cmd/clusterctl/clusterdeployer/clusterdeployer.go

```diff
@@ -404,30 +404,30 @@ func getClusterAPIObject(client clusterclient.Client, clusterName, namespace str
 		return nil, nil, nil, fmt.Errorf("unable to fetch cluster %v in namespace %v: %v", clusterName, namespace, err)
 	}
 
-	master, nodes, err := extractMasterMachine(machines)
+	controlplane, nodes, err := extractControlPlaneMachine(machines)
 	if err != nil {
-		return nil, nil, nil, fmt.Errorf("unable to fetch master machine in cluster %v in namespace %v: %v", clusterName, namespace, err)
+		return nil, nil, nil, fmt.Errorf("unable to fetch controlplane machine in cluster %v in namespace %v: %v", clusterName, namespace, err)
 	}
-	return cluster, master, nodes, nil
+	return cluster, controlplane, nodes, nil
 }
 
-// extractMasterMachine separates the master (singular) from the incoming machines.
+// extractcontrolplaneMachine separates the controlplane (singular) from the incoming machines.
 // This is currently done by looking at which machine specifies the control plane version
-// (which implies that it is a master). This should be cleaned up in the future.
-func extractMasterMachine(machines []*clusterv1.Machine) (*clusterv1.Machine, []*clusterv1.Machine, error) {
+// (which implies that it is a controlplane). This should be cleaned up in the future.
```

Comment:
I'd just delete the parenthetical; now that the text uses control plane I don't think it adds any value.